### PR TITLE
Queue refactor

### PR DESCRIPTION
**Pull request by Ferdinand737**
# Description

Student and TA queue have been combined into one queue

Question tag is displayed on question

### CODE REVIEW

**Comment by AdamFipke** [comment]
I like it! I think it's good to merge in.

Not gonna approve it since I'm still new to the code, @d3li0n @wskksw if one of y'all wanna do a code review

File: packages/app/components/Queue/Queue.tsx [hunk 1]
```diff
@@ -0,0 +1,583 @@
...
+  QuestionStatusKeys,
+  Role,
+} from '@koh/common'
+import { useTAInQueueInfo } from '../../hooks/useTAInQueueInfo'
+import { useCourse } from '../../hooks/useCourse'
+import {
+  QueueInfoColumn,
+  QueueInfoColumnButton,
+} from './QueueListSharedComponents'
+import { Card, Col, Popconfirm, Row, Tooltip, notification } from 'antd'
```
Comment (d3li0n) [inline packages/app/components/Queue/Queue.tsx:20]: Unused import?

File: packages/app/components/Queue/Queue.tsx [hunk 2]
```diff
@@ -0,0 +1,583 @@
...
+        // TODO: how should we handle error that happens for another reason?
+      }
+    },
+    [mutateQuestions, qid, questions],
+  )
+
+  const helpNext = async () => {
+    await onHelpQuestion(nextQuestion.id)
+    mutateQuestions()
+    // openTeams();
```
Comment (d3li0n) [inline packages/app/components/Queue/Queue.tsx:263]: Remove unused code?

File: packages/app/components/Queue/Queue.tsx [hunk 3]
```diff
@@ -0,0 +1,583 @@
...
+      } catch (e) {
+        if (
+          e.response?.data?.message?.includes(
+            ERROR_MESSAGES.questionController.createQuestion.oneQuestionAtATime,
+          )
+        ) {
+          return false
+        }
+        return true
+        // TODO: how should we handle error that happens for another reason?
```
Comment (d3li0n) [inline packages/app/components/Queue/Queue.tsx:254]: I would probably just console log the error / show error message to user.

**Review by d3li0n — COMMENTED** [review]
Few comments.

**Review by d3li0n — APPROVED** [review]
Pending @wskksw approval, but LGTM

**Review by wskksw — COMMENTED** [review]
For each student ticket/row in a queue, the names are not displayed, I don't think that is intended?

**Review by wskksw — APPROVED** [review]

**Pull request merged**